pkg/util/util.go

```diff
@@ -0,0 +1,31 @@
+// Package util houses various utilities that do not fit anywhere else.
```

Comment:
We should focus on removing those (: 

`util` is ALWAYS bad idea as it ends up being pile of trash for everything we are too lazy to think package name ;p

I think this should be actually part of `run`. We can try contribute to run.Group (can start github issue) or fork our own version, as this is must have.